types/swagger2openapi/index.d.ts

```diff
@@ -0,0 +1,145 @@
+// Type definitions for swagger2openapi 7.0
+// Project: https://github.com/Mermade/oas-kit/tree/main/packages/swagger2openapi
+// Definitions by: rxliuli <https://github.com/rxliuli>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Definitions by: rxliuli <https://github.com/rxliuli>
+/// <reference types="node" />
+
+import { OpenAPIV2, OpenAPIV3 } from 'openapi-types';
+import { Readable } from 'node:stream';
+
+export class S2OError extends Error {}
+
+export interface ConvertBaseOptions {
+    text: string; // Both	If not already a truthy value, will be set to the input text of the conversion
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/swagger2openapi/index.d.ts;
  Line: 14;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.